google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequest.java

```diff
@@ -76,6 +78,22 @@ public CreateTableRequest addSplit(ByteString key) {
     return this;
   }
 
+  /** Add change stream retention period between 1 day and 7 days */
+  public CreateTableRequest addChangeStreamRetention(Duration retention) {
+    Preconditions.checkNotNull(retention);
```

Comment:
should we add precondition to check retention is between 1 and 7?

Reply:
I wasn't sure what the philosophy is. Checking here would allow us to fail faster but if we widen the retention period then we have to change on the client side as well.

Reply:
Per https://google.aip.dev/client-libraries/4223, I think we should rely on the server side validation.